babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -874,6 +874,26 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testCreateTableDateFormatDateOnlyColumnLevelAttributes() {
```

Comment:
nit: Attributes -> Attribute

Reply:
This name is a little confusing to me because it contains "DateFormat", but the test case has no date format in it at all. It seems like maybe you're including that in the name because it's part of a date format PR, but it probably doesn't make sense to have it in there in a general context.

Reply:
Thanks, fixed in https://github.com/googleinterns/calcite/pull/103/commits/759efab9843a94cc2467c3e2d47333e5b1c7ef5a